src/routes/missions.py

```diff
@@ -17,154 +19,6 @@
 router = APIRouter(prefix="/api", tags=["Missions & Progress"])
 
 
-@router.get("/progress", response_model=ProgressResponse)
```

Comment:
why was this removed? not needed or what?

Does this introduce any breaking changes for frontend?